tests/fixtures/README.md

```diff
@@ -0,0 +1,42 @@
+
+Using these fixtures will make creating tests more straight-forward as:
+* It should prevent issues with the plugin version as used in the tests (version of the created zip artifact) not matching the version constraint for the plugin in a _real_ standard, which would result in Composer downloading from Packagist instead of using the artifact package created for use in the tests.
+* It means we don't need to keep track of the version history of external standards in regards to:
+    - whether or not the external standard uses the correct project `type` in their `composer.json`.
+    - whether or not they `require` the plugin.
+    - whether or not they comply with the PHPCS naming conventions.
+    - whether or not the standard is compatible with PHPCS 2.x/3.x/4.x.
+    - etc...
+
+Each subdirectory in this `fixtures` directory will be zipped up and placed in the `artifact` subdirectory ahead of running the tests, making them available to all tests.
+
+The artifact version of each fake standard will always be `1.0.0`, unless otherwise indicated.
+Setting a different version for a fake standard can be achieved by explicitly setting the `version` in the `composer.json` file of the fake standard.
+
+Any particular test can use one or more of these fake standards.
+
+Notes:
+* The "fake" standards DO need to comply with the naming conventions from PHPCS, which means that the name of a standard as set in the `ruleset.xml` file MUST be the same as the name of the directory containing the `ruleset.xml` file.
+    So the `ruleset.xml` file for a standard called `Dummy` MUST be in a (sub)directory named `Dummy`.
+* A "fake" standard will normally consist of a `composer.json` file in the fake project root and one or more `ruleset.xml` files.
+* If the "fake" standard `require`s the plugin, it should do so with `'*'` as the version constraint.
+* The "fake" standards generally do NOT need to contain any sniffs or actual rules in the ruleset (unless the standard will be used in a test for running PHPCS).
+
+It is recommended to add a short description of the situation which can be tested with each fixture to the below list.
+
+### Package name: `phpcs-composer-installer/dummy-subdir`
+
+**Description:**
+An external PHPCS standard with the `ruleset.xml` file in a subdirectory ("normal" standard setup).
+
+| Characteristics          | Notes                                                                                            |
+|--------------------------|--------------------------------------------------------------------------------------------------|
+| **Standard(s):**         | `DummySubDir`                                                                                    |
+| **Includes sniff(s):**   | :heavy_checkmark: One sniff - `DummySubDir.Demo.Demo` - which is PHPCS cross-version compatible. |
```

Comment:
To make sure the text renders across different tools/stacks, I'd suggest using the actual emoji rather than the word mark.

Reply:
I've never actually heard of problems with the rendering... can you give me an example ?

Reply:
Basically any places that does not do `:..:` parsing, like text-editors, command-line, etc.

![image](https://user-images.githubusercontent.com/195757/152693479-da93607d-a066-4972-90b9-2360898aecd9.png)

Reply:
Ah, but in my mind, that's a good thing as a) not all editors support UTF-8, b) the file may not have been saved as UTF-8 to begin with, which could lead to a corrupted character and c) rendering of emojis on different OS-es or even different websites can be different, so having the original intention in the text by using the `:...:` annotation makes it clear to whomever is editing what the intended emoji is.